app/src/main/java/com/teamagam/gimelgimel/app/message/view/SendMessagesFragment.java

```diff
@@ -81,11 +78,30 @@ public void takePicture() {
         startCameraIntent(newImageUri);
     }
 
+    public void setSendTextFabClickable(boolean value) {
```

Comment:
That's not a big thing, since the method is short (which is good :>), but a value can be renamed to isClickable to promote readability